front/components/assistant_builder/tags/TagSearchInput.tsx

```diff
@@ -68,7 +85,7 @@ export const TagSearchInput = ({
           </DropdownMenuTrigger>
           <DropdownMenuPortal>
             <DropdownMenuContent align="start">
-              <ScrollArea className="max-h-64">
+              <ScrollArea>
```

Comment:
Any reason we got rid of the max height?

Reply:
Yes but not a good one, my scroll area was not scrolling and I decided to temporarily remove the fixed height! I need to add it back!